### DISCUSSION TITLE
可否支持文章保存到S3，目前仅支持附件保存到S3

### DISCUSSION THREAD

**Discussion by FunkyGod** [halo-dev/halo#5588] [Ideas]
### Your current Halo version

2.12.0

### Describe this feature

希望支持文章保存到S3，目前仅支持附件保存到S3

**Comment by ruibaby** [comment]
当前文章是持久化到数据的，包含多项字段和其他模型的关联，应该很难把每一篇文章当做一个文件一样上传到 S3。这样做的使用场景是什么呢？

**Comment by guqing** [comment]
你可以选择自己写个插件来实现这种没什么普适性的功能

**Comment by wan92hen** [comment]
感觉这个功能的前提，是先支持生成静态页面，Halo 相当于一个带后台的 静态页面生成器
